Lib/test/test_plistlib.py

```diff
@@ -225,7 +225,9 @@ def test_indentation_dict_mix(self):
 
     def test_uid(self):
         data = UID(1)
-        self.assertEqual(plistlib.loads(plistlib.dumps(data, fmt=plistlib.FMT_BINARY)), data)
+        for fmt in ALL_FORMATS:
```

Comment:
Maybe add another test case to verify dict format is as expected? (existing tests just assert that loads/dumps are inverses of each other, not necessarily the correct format)

Reply:
Makes sense. I guess the existing tests are like that because nobody wants to write XML 😅